llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -732,17 +773,25 @@ def tensor_hook(
                         monitor_process = None
 
                     # Spawn a new process to register the model.
+                    # Slower method to register the model via log_model.
                     process = SpawnProcess(
-                        target=_register_model_with_run_id_multiprocess,
+                        target=_log_model_multiprocess,
                         kwargs={
                             'mlflow_logger':
                                 mlflow_logger,
-                            'composer_logging_level':
-                                logging.getLogger('composer').level,
-                            'model_uri':
-                                local_save_path,
-                            'name':
+                            'transformers_model_path':
+                                temp_save_dir,
+                            'flavor':
+                                'peft' if self.using_peft else 'transformers',
```

Comment:
we haven't added peft support for `log_model` in Composer yet. Will need to think about how to handle this.

Reply:
What do we do for peft models then? Do I need to fork the old model registration logic for PEFT models until we find a solution?

Reply:
I think you can just leave this flavor as transformers only...the peft logging API is marked as experimental in Composer.

Reply:
Wouldn't I need the original register_model call for PEFT models though?

Reply:
to preserve the existing behavior, yes. But it was marked as experimental and I'm pretty sure no one used it, and we will redo it now that mlflow supports peft model logging directly.